tests/utils/landing_state.ts

```diff
@@ -0,0 +1,148 @@
+import { sort } from "fast-sort";
+import path from "path";
+import { FileIssue, LandingState, LintAction, TaskFailure } from "tests/types";
+import { REPO_ROOT } from "tests/utils";
+
+const extractFIFields = ({
```

Comment:
```suggestion
const extractFileIssueFields = ({
```

same throughout this file- in general unless the brevity is absolutely needed for readability, go for verbosity